core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestratorSpec.scala

```diff
@@ -610,107 +556,53 @@ class BillingProjectOrchestratorSpec extends AnyFlatSpec {
         e.jobType == BpmBillingProjectDelete
     val monitorRecordDao = mock[WorkspaceManagerResourceMonitorRecordDao](RETURNS_SMART_NULLS)
     when(monitorRecordDao.create(ArgumentMatchers.argThat(matchedExpectedEvent))).thenReturn(Future.successful())
-    val billingProfileManagerDAO = mock[BillingProfileManagerDAO]
 
-    val bpo = new BillingProjectOrchestrator(
-      testContext,
-      alwaysGiveAccessSamDao,
-      mock[NotificationDAO],
-      billingProfileManagerDAO,
-      happyBillingRepository(Some(azureBillingProfile), billingProfileManagerDAO),
-      mock[BillingProjectLifecycle](RETURNS_SMART_NULLS), // google
-      initiateDeleteLifecycle(Future.successful(Some(jobId))), // azure
-      mock[BillingProjectDeletion],
-      mock[MultiCloudWorkspaceConfig],
-      monitorRecordDao
+    val landingZoneId = UUID.randomUUID()
+    val billingRepository = happyBillingRepository(Some(landingZoneId))
+
+    val azureBillingProjectLifecycle = mock[AzureBillingProjectLifecycle]
+    when(azureBillingProjectLifecycle.deleteJobType).thenReturn(BpmBillingProjectDelete)
+    when(azureBillingProjectLifecycle.cleanupLandingZone(landingZoneId, testContext))
+      .thenReturn(Some(new DeleteAzureLandingZoneResult().jobReport(new JobReport().id(jobId.toString))))
+
+    val bpo = spy(
+      new BillingProjectOrchestrator(
+        testContext,
+        alwaysGiveAccessSamDao,
+        mock[NotificationDAO],
+        billingRepository,
+        mock[GoogleBillingProjectLifecycle](RETURNS_SMART_NULLS),
+        azureBillingProjectLifecycle,
+        mock[BillingProjectDeletion],
+        mock[MultiCloudWorkspaceConfig],
+        monitorRecordDao
+      )
     )
+    doReturn(Future.successful()).when(bpo).maybeDeleteGoogleProject(billingProjectName, testContext)
 
     Await.result(bpo.deleteBillingProjectV2(billingProjectName), Duration.Inf)
 
     verify(monitorRecordDao).create(ArgumentMatchers.argThat(matchedExpectedEvent))
-  }
-
-  it should "not create a job to delete the Azure project after calling maybeCleanupResources fails" in {
```

Comment:
Folded into "fail if cleanupLandingZone throws an exception and not create a job to delete the Azure project".